packages/newspringchurchapp/src/prayer/AddPrayer/AddPrayerCard/index.js

```diff
@@ -0,0 +1,49 @@
+import React, { memo } from 'react';
+import { Image } from 'react-native';
+import PropTypes from 'prop-types';
+import { H5, styled, Button, Card, CardContent } from '@apollosproject/ui-kit';
+import AddPrayerHeader from '../AddPrayerHeader';
+
+const StyledCardContent = styled(() => ({
+  alignItems: 'center',
+}))(CardContent);
+
+const Description = styled(({ theme }) => ({
+  color: theme.colors.text.secondary,
+  marginBottom: theme.sizing.baseUnit,
+}))(H5);
+
+const AddPrayerButton = styled(({ theme }) => ({
+  marginTop: theme.sizing.baseUnit * 2,
+  width: '100%',
+}))(Button);
+
+const ButtonLabel = styled(({ theme }) => ({
+  color: theme.colors.text.primary,
+}))(H5);
+
+const AddPrayerCard = memo(({ imgSrc, description }) => (
+  <Card>
+    <StyledCardContent>
```

Comment:
Could probably add a `<PaddedView>` in between `<Card>` and `<StyledContentCard>` but honestly, I'd like to see us modularize this since there are at least 3 different places where we use very similar UI. I'm keeping a list of polish items ... I just need to make a task to list them all.

Reply:
love that! I'll leave this here and let you polish it all up later, make sure to add to that list "organize prayer stuff"